router/courses.js

```diff
@@ -351,7 +304,7 @@ router.post('/courses/enroll/', function (request, response) {
     var classid = request.body.cid;
```

Comment:
classId is courseOfferinId?

Reply:
If I remember correctly, it should be courseOfferingId.

Reply:
Then, change all the variables to courseOfferingId

Reply:
resolved